soco/core.py

```diff
@@ -1369,6 +1369,9 @@ def add_to_queue(self, queueable_item, position=0, as_next=False):
                 added. Default is 0 (add URI at the end of the queue).
             as_next (bool): Whether this URI should be played as the next
                 track in shuffle mode. This only works if `play_mode=SHUFFLE`.
+                
```

Comment:
The blank line contains a whitespace, which flake8 doesn't like. Could you please remove it?